src/AppServices/KafkaPage/KafkaPage.tsx

```diff
@@ -35,50 +49,212 @@ export const KafkaPage: FunctionComponent = () => {
+                      }}
+                    />
+                  </Text>
+                </TextContent>
+              </FlexItem>
+            </Flex>
+          </CardBody>
+        </Card>
+      </MarketingPageSection>
+      <MarketingPageSection kafkaOverviewPage={true}>
+        <Card>
+          <CardTitle>
+            <Title headingLevel="h2">{t("kafkaInstanceCapacityTitle")}</Title>
+          </CardTitle>
+          <CardBody>
+            <Stack hasGutter>
+              <Text component={TextVariants.p}>
+                {t("kafkaInstanceCapacityDescription")}
+              </Text>
+              <KafkaInstanceCapacityTable />
+            </Stack>
+          </CardBody>
+          <CardFooter>
+            <TextContent className={"pf-u-font-size-sm"}>
+              {t("kafkaInstanceCapacityfooter1")}{" "}
+              <Button
+                data-testid="redhatSupport-buttonCTA"
+                isInline
+                variant={ButtonVariant.link}
+                component="a"
+                target="_blank"
+                href="https://access.redhat.com/support"
+              >
+                {" "}
+                Redhat support
+                <ExternalLinkAltIcon className="pf-u-ml-xs" />
+              </Button>{" "}
+              {t("kafkaInstanceCapacityfooter2")}
+              <Button
+                data-testid="diskSpaceMetrics-buttonCTA"
+                isInline
+                variant={ButtonVariant.link}
+                component="a"
+                target="_blank"
+                href="https://access.redhat.com/documentation/en-us/red_hat_openshift_streams_for_apache_kafka/1/guide/aced8e5e-8229-4cb2-82f9-87a8caa24bb3"
+              >
+                {" "}
+                monitoring disk space metrics
+                <ExternalLinkAltIcon className="pf-u-ml-xs" />
+              </Button>
+              .
+            </TextContent>
+          </CardFooter>
+        </Card>
+      </MarketingPageSection>
+      <MarketingPageSection kafkaOverviewPage={true}>
+        <Card>
+          <CardTitle>
+            <Title headingLevel="h2">{t("cloudProvidersTitle")}</Title>
+          </CardTitle>
+          <CardBody>
+            <Flex spaceItems={{ default: "spaceItemsXl" }}>
+              <Flex
+                alignSelf={{ default: "alignSelfCenter" }}
+                justifyContent={{ default: "justifyContentCenter" }}
+                flex={{ default: "flex_1" }}
+              >
+                <FlexItem>
+                  <Stack>
+                    <StackItem>
+                      <img src={LogoAWS} alt={""} style={{ height: "60px" }} />
+                    </StackItem>
+                    <StackItem>
+                      <Title headingLevel="h2" className={"pf-u-pt-sm"}>
+                        {"Amazon Web Services"}
+                      </Title>
+                    </StackItem>
+                  </Stack>
+                </FlexItem>
```

Comment:
The layout of the images and text is slightly off. There are a couple of options as shown in this design that we could implement.

![image](https://user-images.githubusercontent.com/21063328/165850951-83472941-edda-4182-8468-43cefc4a1d60.png)

The first option involves replacing the `<Stack>` and `<StackItem>` components with `<Split hasGutter>` and `<SplitItem>`.

The second option involves removing the Stack layout and using the parent Flex layout to align the contents to be centered. I think you do this with `align={{ default: 'alignCenter' }}`, but I could be wrong. You would also need to check the space between elements and use some of the space between props to adjust the spacing. There are props you can apply on the Flex parent, but there are also props you can specify on the FlexItem to modify just the spacing for that element.